packages/fiber/tests/core/renderer.test.tsx

```diff
@@ -845,11 +845,6 @@ describe('renderer', () => {
     // Creates an isolated state enclave
     expect(state.scene).not.toBe(scene)
     expect(portalState.scene).toBe(scene)
-
-    // Preserves internal keys
-    const overwrittenKeys = ['get', 'set', 'events', 'size', 'viewport']
-    const respectedKeys = privateKeys.filter((key) => overwrittenKeys.includes(key) || state[key] === portalState[key])
-    expect(respectedKeys).toStrictEqual(privateKeys)
```

Comment:
This never worked since the store was polluted on creation. Don't need to change behavior either.